### PR TITLE
Feat/external key validation docker config

### PR DESCRIPTION
**Pull request by czervenka**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
-__version__ = "0.10.0-rc1"
```
Comment (xaralis) [inline docs/source/conf.py:17]: This probably isn't intentional, right?

File: src/fakeprinter/fakeprinter/__init__.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from flask_cors import CORS, cross_origin
 from werkzeug.utils import secure_filename
 
-__version__ = "0.10.0-rc1"
```
Comment (xaralis) [inline src/fakeprinter/fakeprinter/__init__.py:11]: Same here.

File: src/karmen_backend/Dockerfile [hunk 3]
```diff
@@ -48,6 +48,7 @@ ENV POSTGRES_DB_USER print3d
 
 ENV SERVICE_HOST 0.0.0.0
 ENV SERVICE_PORT 9764
+ENV TOKEN_SERVER_API_URL ''
```
Comment (xaralis) [inline src/karmen_backend/Dockerfile:51]: Damn, I'm sorry to have forgotten that one.

File: src/karmen_backend/server/__init__.py [hunk 4]
```diff
@@ -8,7 +8,7 @@
 from flask_executor import Executor
 from sentry_sdk.integrations.flask import FlaskIntegration
 
-__version__ = "0.10.0-rc1"
+__version__ = "v0.9.0-rc18"
```
Comment (xaralis) [inline src/karmen_backend/server/__init__.py:11]: Same here.

File: src/karmen_frontend/package-lock.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "karmen",
-  "version": "0.10.0-rc1",
```
Comment (xaralis) [inline src/karmen_frontend/package-lock.json:3]: .. and here

File: src/karmen_frontend/package.json [hunk 6]
```diff
@@ -45,7 +45,7 @@
       "plugin:prettier/recommended"
     ]
   },
-  "version": "0.10.0-rc1",
+  "version": "v0.9.0-rc18",
```
Comment (xaralis) [inline src/karmen_frontend/package.json:48]: And here? Cause we have 0.9.0 rc already deployed in prod so I think we're now in process of preparing 0.10.0, right?

File: docs/source/conf.py (hunk 1 above)
Comment (czervenka) [reply docs/source/conf.py:17]: it's not, I didn't notice the tag in git

**Pull request merged**